packages/eslint-plugin-wantedly/src/rules/nexus-camel-case-field-name.ts

```diff
@@ -6,7 +6,7 @@ import { docsUrl, getOptionWithDefault, isNexusSchemaImported } from "./utils";
 const linter = new Linter();
 export const RULE_NAME = "nexus-camel-case-field-name";
 
-const WHITELIST_FOR_TYPE_DEFINITION = ["objectType", "interfaceType", "inputObjectType"];
+const ALLOWLIST_FOR_TYPE_DEFINITION = ["objectType", "interfaceType", "inputObjectType"];
```

Comment:
The function names not in this array are **not all denied.** At this, these function names are mean to corresponding to this rule. So **allow / deny is not appropriate** in this situation.

Could you rename this to `TYPE_DEFINITION_METHODS`?